cpu/esp32/bootloader/sdkconfig.h

```diff
@@ -85,7 +85,25 @@ extern "C" {
  * Bootloader output baudrate, defined by the app settings as BAUD or
  * BOOTLOADER_BAUD.
  */
-#define CONFIG_ESP_CONSOLE_UART_BAUDRATE (RIOT_BOOTLOADER_BAUD)
+#define CONFIG_ESP_CONSOLE_UART_BAUDRATE    (RIOT_BOOTLOADER_BAUD)
+
+/* 
```

Comment:
```suggestion
/*
```

Reply:
I was afraid of that :worried: Let me check.

Reply:
I'm not sure whether the reconfiguration of UART make sense at all for ESP8266. It has only one full functional UART interface. The TRM for ESP8266 is so pure, that I don't know whether the signals are hardwired or routable. Futhermore,  I don't know whether there is a ESP8266 board in universe that doesn't have connected the UART0 to the UART-to-USB bridge. So I would remove the configuration of GPIOs from `uart_init`.